x-pack/plugins/lens/public/datasources/form_based/field_list.tsx

```diff
@@ -1,220 +0,0 @@
-/*
```

Comment:
It was transformed into `src/plugins/unified_field_list/public/components/field_list/field_list_grouped.tsx`